TAG_SYNTAX.md

```diff
@@ -0,0 +1,113 @@
+# Tag Syntax
+
+An example from the Java performer, where an expiry() overload was added in version 3.0.7:
+```
+// [if:3.0.7]
+out.expiry(Instant.ofEpochSecond(opts.getExpiry().getAbsoluteEpochSecs()));
+// [else]
+throw new UnsupportedOperationException(
+    "This SDK version does not support absolute expiry"
+);
+// [end]
+```
+
+The `else` clause is optional, so this is also valid:
+```
+// [if:3.0.7]
+somethingThatRequiresThisVersion();
+// [end]
+```
+
+## Version comparison operators
+
+The default version comparison operator is "greater than or equal to".
+You can also do a "less than" comparison:
+```
+// [if:<3.0.7]
+somethingForEarlierVersions();
+// [end]
+```
+
+## Version ranges
+
+You can "and" two conditions to express a version range:
```

Comment:
Nice!

Reply:
*You* did this! I just documented it :-D